UnitTests/UnitTests/ProjectBLTests.cs

```diff
@@ -72,8 +102,61 @@ public static Mock<UserManager<ApplicationUser>> MockUserManager(List<Applicatio
         }
 
         [TestMethod]
-        public void TestMethod1()
+
+		public void TestGetProject()
+		{
+			
+			int projectId = 1;
+
+			Project project = ProjectBusinessLogic.GetProject(projectId);
+
+
+			Assert.AreEqual(projectId, project.Id);
+			
+		}
+
+        [TestMethod]
+        public void TestGetProject_IdIsNull()
         {
-        }
-    }
+			int? projectId = null;
+
+			Project project = ProjectBusinessLogic.GetProject(projectId);
+
+            Assert.IsNull(project);
+		}
+
+        [TestMethod]
+
+        public void TestDeleteProject()
+        {
+            int projectIdToDelete = 1;
+
+			int initialProjectCount = projectData.Count();
+
+			Project deletedProject = ProjectBusinessLogic.DeleteProject(projectIdToDelete);
+
+			projectData = projectData.Where(p => p.Id != projectIdToDelete);
+
+			int updatedProjectCount = projectData.Count();
+
+			Assert.AreEqual(projectIdToDelete, deletedProject.Id);
+
+			Assert.AreEqual(initialProjectCount - 1, updatedProjectCount);
+
+			bool isProjectDeleted = !projectData.Any(p => p.Id == projectIdToDelete);
+
+			Assert.IsTrue(isProjectDeleted);
+
+		}
+
+		[TestMethod]
```

Comment:
Test Method names should be more detailed.
ex.
"DeleteProject_ProjectNotFound_ThrowsKeysNotFound()"